tables/tests/test_vlarray.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: latin-1 -*-
-
 import sys
```

Comment:
Please revert this.
I have had a lot of problems in the past for that coding cookie :/

Reply:
Wouldn't it be safer to replace those 17 occurrences of the `REPLACEMENT CHARACTER` with `\ufffd` or `\N{REPLACEMENT CHARACTER}` or a variable?

Reply:
WIP in #839